internal/compiler-bridge/src/main/scala/xsbt/InteractiveConsoleFactoryBridge.scala

```diff
@@ -25,16 +26,16 @@ class InteractiveConsoleBridgeFactory extends xsbti.InteractiveConsoleFactory {
       bindNames: Array[String],
       bindValues: Array[AnyRef],
       log: Logger
-  ): xsbti.InteractiveConsoleInterface =
-    new InteractiveConsoleBridge(
-      args,
-      bootClasspathString,
-      classpathString,
-      initialCommands,
-      cleanupCommands,
-      loader,
-      bindNames,
-      bindValues,
-      log
-    )
+  ): xsbti.InteractiveConsoleInterface = new InteractiveConsoleBridge(
+    args,
+    bootClasspathString,
+    classpathString,
+    initialCommands,
+    cleanupCommands,
+    loader,
+    bindNames,
+    bindValues,
+    log
+  )
```

Comment:
Not a fan of this dangling.

Reply:
I assume you mean the `new InteractiveConsoleBridge(` folding up.  See the comment higher about the options here.